pkg/providers/instancetype/skus.go

```diff
@@ -22,10 +22,10 @@ import (
 	"k8s.io/utils/ptr"
 )
 
-// getKarpenterWorkingSKUs returns a the list of SKUs that are
+// GetKarpenterWorkingSKUs returns a the list of SKUs that are
 // allowed to be used by Karpenter. This is a subset of the
 // SKUs that are available in Azure.
-func getKarpenterWorkingSKUs() []skewer.SKU {
+func GetKarpenterWorkingSKUs() []skewer.SKU {
 	workingSKUs := []skewer.SKU{}
 	for _, sku := range allAzureVMSkus {
```

Comment:
Does the `allAzureVMSkus` need to be updated? Is it automatically tested for updates in our CI somehow?

Reply:
Not updated automatically yet, but will be in the future.